modules/preparing-for-installing-microshift-rpm.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc 
+
+[id="preparing-install-microshift-from-rpm-package_{context}"]
+= Preparing to install MicroShift from an RPM package
+
+Before installing MicroShift from an RPM package, you must configure your {op-system} machine to have a logical volume manager (LVM) volume group (VG) with sufficient capacity for the persistent volumes (PVs) of your workload.
+
+MicroShift uses the logical volume manager storage (LVMS) Container Storage Interface (CSI) provider for storing PVs. LVMS relies on Linux's LVM to dynamically manage the backing storage for PVs. For this reason, your machine must have an LVM VG in which LVMS can create the LVM logical volumes (LVs) for your workload's PVs.  
+
+The simplest way to configure an LVM VG that allows LVMS to create the LVM LVs for your workload's PVs is to set your machine to reserve capacity during the installation of {op-system}. 
```

Comment:
"reserve capacity during the installation" is unclear to me?

Reply:
I suspect this refers to the `spare-gb` field provided by the lvmd api.   After some discussion, I believe we've decided to make this field opaque to users to avoid the issues surrounding the reservation logic.